ui/views/view.cc

```diff
@@ -1840,6 +1840,12 @@ void View::PropagateNativeThemeChanged(const ui::NativeTheme* theme) {
   OnNativeThemeChanged(theme);
 }
 
+void View::PropagateSoftVisibilityChanged(bool visible) {
+  for (int i = 0, count = child_count(); i < count; ++i)
```

Comment:
do we really win anything caching 'count'? for (int i = 0; i < child_count(); ++i) ?